solidity/contracts/isms/native/AbstractNativeISM.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity >=0.8.0;
+
+// ============ Internal Imports ============
+
+import {IInterchainSecurityModule} from "../../interfaces/IInterchainSecurityModule.sol";
+import {Message} from "../../libs/Message.sol";
+import {TypeCasts} from "../../libs/TypeCasts.sol";
+
+// ============ External Imports ============
+
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+
+/**
+ * @title ArbtractNativeISM
+ * @notice Uses the native bridges to verify interchain messages.
+ */
+abstract contract AbstractNativeISM is IInterchainSecurityModule, Ownable {
```

Comment:
What is ownable here?

Reply:
It's used by the ISMs for setOptimismHook.